classes/event/user_locked.php

```diff
@@ -44,13 +44,17 @@ class locked_out extends \core\event\base {
      * Create instance of event.
      *
      * @param int $userid the userid of the User whos account was locked
+     * @param int $tier the tier the user just moved onto.
      * @return locked_out the locked out event
      */
-    public static function locked_out_event($user) {
+    public static function locked_out_event($user, $tier) {
 
         $data = array(
             'context' => \context_user::instance($user->id),
-            'other' => array ('userid' => $user->id)
+            'other' => array (
+                'userid' => $user->id,
+                'tier' => $tier
```

Comment:
I think this should have the absolute time they are locked out until, the tier could change over time is a little obscure by itself

Reply:
Changed to absolute time in the logs.